### PR TITLE
Refactor constants and packages

### PR DESCRIPTION
**Pull request by mdelolmo**

### CODE REVIEW

**Comment by mdelolmo** [comment]
@veronikapetruskova

File: auth-lib/src/main/java/com/spotify/sdk/android/auth/app/SpotifyNativeAuthUtil.java [hunk 1]
```diff
@@ -30,46 +30,30 @@
...
-    private static final String EXTRA_VERSION = "VERSION";
+public class SpotifyNativeAuthUtil {
 
     /*
      * Constants below have their counterparts in Spotify app where
-     * they're used to parse messages received from SDK. 
+     * they're used to parse messages received from SDK.
      * If anything changes in the structure of those messages this version
-     * should be bumped and new protocol version should be implemented on the other side. 
+     * should be bumped and new protocol version should be implemented on the other side.
```
Comment (veronikapetruskova) [inline auth-lib/src/main/java/com/spotify/sdk/android/auth/app/SpotifyNativeAuthUtil.java]: this comment should be moved with the constants, right?

Comment (mdelolmo) [reply auth-lib/src/main/java/com/spotify/sdk/android/auth/app/SpotifyNativeAuthUtil.java]: The first part yes, I will clean up or rephrase

**Comment by mdelolmo** [comment]
@veronikapetruskova I rephrased and moved the comment to `IntentExtras`.

**Pull request merged**